projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -28,7 +28,7 @@
 
   <ItemGroup>
     <Compile Remove="build\**\*" />
-    <Compile Include="..\..\..\gensrc\autogenerated-api-0-9-1.cs" Exclude="build\**\*;bin\**;obj\**;**\*.xproj;packages\**" />
```

Comment:
Why was this line changed?

Reply:
To allow me to compile the project.
Included the change to allow a diff on the output from the templating.
The change makes a class partial to allow inclusion of async overloads in another file.
I get why your doing this, but is it really necessary?